### PR TITLE
List the services belonging to an active package

### PR DESCRIPTION
**Pull request by stevenwilkin**
**WebDM** links to web UIs provided by active packages.

This PR facilitates this by providing the details of any services belonging to a specific package, including port information.

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by mvo5** [comment]
Thanks for this branch. Looks good! 

The future of the "ports" part of the service struct is uncertain, after the last sprint there was discussion that it will go away in favour of capabilities.  However this does not affect this branch, once this move happens we simply do it here too :)

**Comment by stevenwilkin** [comment]
I accepted that I was working against a moving target with this @mvo5, hopefully I'll be able to keep up! Cheers for the feedback once again :smile:

**Comment by chipaca** [comment]
:+1: although I guess the merge needs to wait for the refactor/rebase

**Comment by stevenwilkin** [comment]
Cheers for the :+1:s chaps. I'll have the refactor piece done later today.

**Comment by stevenwilkin** [comment]
Rebased on master to make use of #285

File: client/services.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	PackageName     string `json:"package_name"`
+	ServiceName     string `json:"service_name"`
+}
+
+// Services returns the list of services belonging to an *active* Package
+func (client *Client) Services(pkg string) (map[string]Service, error) {
+	var services map[string]Service
+
+	path := fmt.Sprintf("/1.0/packages/%s/services", pkg)
+	if err := client.doSync("GET", path, nil, &services); err != nil {
```
Comment (mvo5) [inline client/services.go]: So nice how simple this now looks!

**Comment by mvo5** [comment]
:+1:

File: client/services.go (hunk 1 above)
Comment (stevenwilkin) [inline client/services.go]: It clears things up a great deal, cheers for providing the guts of the refactor :)

**Pull request merged**